src/bugpoint.rs

```diff
@@ -67,8 +67,9 @@ pub fn run(
 }
 
 enum MutationKind {
-    /// The mutation reduced the amount of instructions or ebbs.
-    Shrinked,
+    /// The mutation raised or reduced the amount of instructions or ebbs.
+    ExpandedOrShrinked,
```

Comment:
Agreed the name I've chosen isn't the best. However, renaming this "Progress" would be confusing with the below "Changed", which ought to be renamed too.

Maybe this whole enum's name should be about the size, so something like "SizeEffect" instead of "MutationKind", "ExpandedOrShrinked" => "Changed", "Changed" => "None".

Reply:
(I'll defer this for the moment.)